pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -37,8 +37,9 @@ import "cloud/cloudpb/external_storage.proto";
 // descriptor in the database, and doesn't emit any rows nor support
 // any post-processing.
 message BackfillerSpec {
+  // TODO(lidor): this is not needed in 23.2, the new way to send the job tag is
```

Comment:
minor nit: rather than "in 23.2", I'd say "when interoperability with 22.2  is dropped" or something, which may or may not be 23.2 but we'd be trying to predict the future if we specify which version will drop backwards compat with 22.2.

Reply:
Done, thanks!